prime-router/metadata/hl7_mapping/ORU_R01/base/contact.yml

```diff
@@ -0,0 +1,64 @@
+name: ORU-R01-contact
+elements:
+  - name: contact-family-name
+    condition: '%resource.name.family.exists()'
```

Comment:
In this specific case, you do not need the condition as if there is no value then the field is set to blank.